rest/route/middleware.go

```diff
@@ -599,6 +600,13 @@ func updateHostAccessTime(ctx context.Context, h *host.Host) {
 // canAlwaysSubmitPatchesForProject returns true if the user is a superuser or project admin,
 // or is authorized specifically to patch on behalf of other users.
 func canAlwaysSubmitPatchesForProject(user *user.DBUser, projectId string) bool {
+	if projectId == "" {
+		grip.Error(message.Fields{
```

Comment:
nit: you can simplify this down to `grip.ErrorWhen(projectId == "", ...)`

Reply:
Oh I meant to also return here since the follow up lines are useless